Resources/Prototypes/_Scp/theme.yml

```
@@ -0,0 +1,17 @@
+﻿- type: uiTheme
+  id: ScpTheme
+  path: /Textures/_SCP/Interface/Interface/
```

```suggestion
  path: /Textures/_Scp/Interface/ScpTheme
```

Интерфейс/Интерфейс выглядит убого, переименуй папку в `ScpTheme` + исправь ошибку в регистре Scp